lib/hanami/action/response.rb

```diff
@@ -72,7 +72,7 @@ def body=(str)
       end
 
       def render(view, **options)
-        self.body = view.(**view_options.(request, self), **options).to_str
+        self.body = view.(**view_options.(request, self), **options.merge(exposures)).to_str
```

Comment:
Regarding your notes in the main PR description:

> I'm not sure if options.merge(exposures) is the best way to do, because it will override explicit options in case of keys clash.
>
> ```ruby
> res[:foo] = "bar"
> res.render(view, foo: "baz")
> ```
> 
> With the current implementation :foo will return "bar" value. Do we care about this? How shall we solve it?

Could we change this to `**exposures.merge(options)`?

This way the exposures will be the implicit "base" for the options passed to the view render, and any _explicit_ options – the actual arguments passed to the view's `#call` – will take precedence, overriding the exposures in the case of matching keys, which feels like the right order to me.

What do you think @jodosha?

Reply:
@timriley Thanks I had the same idea, happy that we're on the same page. 😃

Reply:
Fixed by 6c53dd97e4ebbf3fa0403e4a7050637876c40986